src/renderer/i18n/en/app.json

```diff
@@ -429,8 +429,10 @@
       },
       "sort": {
         "title": "Sort",
-        "name": "Name",
-        "marketcap": "Marketcap"
+        "name_asc": "Name ASC",
+        "name_desc": "Name DESC",
+        "marketcap_asc": "Marketcap ASC",
+        "marketcap_desc": "Marketcap DESC"
```

Comment:
do we need the one from "smaller to highest"?

Reply:
@bscholving your call as far as I'm concerned